dist/api.d.ts

```diff
@@ -0,0 +1,68 @@
+import * as vscode from "vscode";
```

Comment:
We shouldn't need to check in anything from the `dist` directory because it will be recreated by the build.

Reply:
Adding this to the .gitignore.